js/search-algorithm.js

```diff
@@ -136,7 +136,7 @@ define([
 
 		onDataReady: function() {
 			var config = Adapt.course.get("_search");
-			if (!config || !config._isEnabled ) return;
+			if (!config || _.has(config, '_isEnabled') && config._isEnabled === false) return;
```

Comment:
I can change that, but regarding the other checks the `_isEnabled` property is also defined at the contentobject, article, block and component level.